### ISSUE TITLE
Extreme Damage Bug! (BB #1031)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1031]
This issue was migrated from bitbucket.
**Original Reporter:** Kelzior
**Original Date:** 30.06.2015 15:19:06 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1031

<hr>

You simple can stack up the DMG of every class like shaman (melee) or ex. warlock weapon with +40 spell dmg enchant..)

Fill your bags with weapons, die and release at spirit healer all the time till your weapons get red (durability). Apply rockbiter weapon (shaman spell) to the weapon, repair only the equiped weapon, than swap the weapon with another one from your bags, apply rockbiter and repair it. This will increase your melee damage (doing this for like 30min-1h makes extreme damage (millions) can solo bosses)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.06.2015 15:20:42 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1031/#comment-19400112

<hr>

Can I have a screenshot or a quick video showing this? What is it auras stacking or just stats? I need to see something!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.06.2015 17:01:10 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1031/#comment-19402435

<hr>

[Core/Spells] Fix an exploit regarding enchantments, weapons and durability.
              Fixes #1031.

```
          Thanks for reporting and assisting with testing cases Badummtzz,
          Kelo, Edaa.
```

→ <<cset 73f984ff2ed3>>
